mmseg/models/backbones/mit.py

```diff
@@ -0,0 +1,434 @@
+import math
+import warnings
+from functools import partial
+
+import torch
+import torch.nn as nn
+from mmcv.cnn import (ConvModule, build_activation_layer, build_norm_layer,
+                      constant_init, normal_init, trunc_normal_init)
+from mmcv.cnn.bricks.drop import build_dropout
+from mmcv.cnn.bricks.transformer import MultiheadAttention
+from mmcv.runner import BaseModule, ModuleList, Sequential, _load_checkpoint
+
+from ...utils import get_root_logger
+from ..builder import BACKBONES
+from ..utils import PatchEmbed, mit_convert
+
+
+def nlc_to_nchw(tensor, H, W):
+    assert len(tensor.shape) == 3
+    B, _, C = tensor.shape
```

Comment:
```suggestion
    B, L, C = tensor.shape
```
and assert fo L and HW